soc/v2/nuvoton/npcx/npcx4/Kconfig.soc

```diff
@@ -0,0 +1,29 @@
+# Nuvoton NPCX4 EC series
+
+# Copyright (c) 2023 Nuvoton Technology Corporation.
+# SPDX-License-Identifier: Apache-2.0
+
+config SOC_SERIES_NPCX4
+	bool
+	select SOC_FAMILY_NPCX
```

Comment:
The `select` here requires that those lines:
https://github.com/zephyrproject-rtos/zephyr/blob/701375bb69e71dd39fa90d58696a6e8ce6bb289d/soc/v2/nuvoton/npcx/Kconfig#L6-L7
are placed and sourced into the Kconfig.soc tree for hwmv2.

Reply:
fixed